database_wrapper.py

```diff
@@ -607,10 +666,9 @@ def get_course_properties(self, courseName) -> List[str]:
         """
 
         # FIXME: resolve unused variable `props`, until then, commented out
-        props = self.get_property_from_entity(["*"],
-                                              "Courses",
-                                              condition_field="courseName",
-                                              condition_value=courseName)
+        props = self.get_property_from_entity(
+            ["*"], "Courses", condition_field="courseName", condition_value=courseName
+        )
```

Comment:
Should this be commented out as indicated in the FIXME?

Reply:
@snekiam  good point, I'll make another PR of refactorings for dead code, FIXMEs & TODOs